src/components/DeliverableView/DocumentsList.tsx

```diff
@@ -1,10 +1,59 @@
-import { Box } from '@mui/material';
+import { useState } from 'react';
+import { Box, Grid } from '@mui/material';
+import strings from 'src/strings';
+import { SearchSortOrder } from 'src/types/Search';
+import { useLocalization } from 'src/providers';
+import { TableColumnType } from 'src/components/common/table/types';
+import { BaseTable as Table } from 'src/components/common/table';
+import { SortOrder } from 'src/components/common/table/sort';
 import { ViewProps } from './types';
 
+const columns = (activeLocale: string | null): TableColumnType[] =>
```

Comment:
We have a different set of columns for the accelerator view. The props has an `isAccelerator` boolean.
If it's better to pass the columns and renderer into the props, we can do that.. or just handle the entire context here leveraging isAccelerator.

Reply:
This is in #2214